layer/nrlf/core/dynamodb/repository.py

```diff
@@ -238,9 +239,28 @@ def search(
             patient_sort = f"C#{category_id}#T#{type_id}"
             key_conditions.append("begins_with(patient_sort, :patient_sort)")
             expression_values[":patient_sort"] = patient_sort
+        else:
```

Comment:
I'm speaking my mind freely here: Would there ever be the case that there is a single type but multiple categories in a search? Should we handle that as an error? Was it imaging that had multiple categories for a same type code? For example looking for xray right arm both procedure and report?

Reply:
I guess maybe multiple categories we wouldn't do in the alpha

Reply:
there's another ticket for multiple categories filter but multiple categories for type isnt gonna be part of alpha because we ran into that issue before and decided not to deal with it for now